RecoLocalTracker/SiPixelRecHits/src/VVIObjF.cc

```diff
@@ -105,33 +105,81 @@ VVIObjF::VVIObjF(float kappa, float beta2, int mode) : mode_(mode) {
   if (mode_ == 0) {
     a_[n - 1] = omega_ * .31830988654751274f;
   }
-  q = -1.;
-  q2 = 2.;
+
+  float q_[n];
```

Comment:
it would be nice to not mix the trailing underscore for locals and data members.

Reply:
What would you suggest? I thought  of `_array` to distinguish them from the local variables they are replacing.

Reply:
leading underscore is against the naming rules. would a trailing cap (e.g. V or A to denote auto/vectorization) work.